### ISSUE TITLE
Support for other container runtimes

### ISSUE THREAD

**Issue by makentenza** [Dynatrace/dynatrace-oneagent-operator#45]
While deploying oneagent on Nodes with a different container runtime than Docker (tested with CRI-O), the process fails:

```
2018-08-23 20:34:37 UTC [ERROR] Failed to verify that the script was launched in a container
2018-08-23 20:34:37 UTC [ERROR] Container IDs returned by Docker CLI: ''
2018-08-23 20:34:37 UTC [ERROR] /proc/self/cgroup contents: '11:cpuset:/kubepods.slice/kubepods-besteffort.slice/kubepods-besteffort-pode9156505_a713_11e8_92eb_fa163e45867e.slice/crio-b643795f081d783a7088093813dfbcd16af061d20a22d15d9892304b7d7ec64b.scope
```

**Comment by esukram** [comment]
Closing this issue, as this is more on the OneAgent itself. The agent will support CRI-O anyways in the near future.

**Comment by makentenza** [comment]
Hey @esukram, can you point me to the OneAgent repository so I can track the support for CRI-O there?

**Comment by aloismayr** [comment]
@makentenza CRI-O support was released with OneAgent 1.163
https://www.dynatrace.com/news/blog/get-automated-full-stack-visibility-into-your-cri-o-kubernetes-containers-beta/

**Comment by makentenza** [comment]
Cool! Thanks @aloismayr
